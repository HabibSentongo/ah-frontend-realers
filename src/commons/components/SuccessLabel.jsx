/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import '../CSS/StatusLabel.scss';
import { Alert } from 'react-bootstrap';
import PropTypes from 'prop-types';

function SuccessLabel({ message }) {
  return (
    <Alert variant="success" className="status-box">
      {message}
    </Alert>
  );
}

SuccessLabel.propTypes = {
  message: PropTypes.string,
};


SuccessLabel.defaultProps = {
  message: 'Success ',
};

export default SuccessLabel;
