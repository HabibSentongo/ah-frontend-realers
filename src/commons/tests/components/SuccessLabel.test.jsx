/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SuccessLabel from '../../components/SuccessLabel';

// test
describe('SuccessLabel Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<SuccessLabel />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
