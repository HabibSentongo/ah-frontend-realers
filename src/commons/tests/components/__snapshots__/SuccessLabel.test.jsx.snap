// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessLabel Component should match the snapshot 1`] = `
<Alert
  className="status-box"
  closeLabel="Close alert"
  show={true}
  transition={[Function]}
  variant="success"
>
  Success 
</Alert>
`;
